### PR TITLE
Comment out the cross-langauge assembly load parts of perf tests.

### PR DESCRIPTION
**Pull request by pharring**
These have been firing so often lately that we haven't had any good performance data in a long time. I'm disabling the scenarios for now and I'll enter an issue to track down the root cause of the problem and re-enable them.

@rchande @heejaechang for review.

### CODE REVIEW

**Comment by heejaechang** [comment]
:+1:

File: src/EditorFeatures/CSharpTest/PerfTests/CSharpPerfFindRefs.xml [hunk 1]
```diff
@@ -24,8 +24,8 @@
       <WaitForCodeMarkers />
       <ExecuteCommand Command="Window.ActivateDocumentWindow" />
 
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="CSharp" />
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="NoBasic" />
+      <!--<VerifyRoslynModulesLoadedStatus ExpectedStatus="CSharp" />
```
Comment (shyamnamboodiripad) [inline src/EditorFeatures/CSharpTest/PerfTests/CSharpPerfFindRefs.xml]: Perhaps we should just change 'VerifyRoslynModulesLoadedStatus' to stop failing in such cases? That way we will only need to change one place when we re-enable this check?

Comment (heejaechang) [inline src/EditorFeatures/CSharpTest/PerfTests/CSharpPerfFindRefs.xml]: seems good idea.

Comment (pharring) [inline src/EditorFeatures/CSharpTest/PerfTests/CSharpPerfFindRefs.xml]: Well, this change isn't all the tests. I left alone the typing tests and the diagnostics tests.

Comment (shyamnamboodiripad) [inline src/EditorFeatures/CSharpTest/PerfTests/CSharpPerfFindRefs.xml]: Ah I see then it makes sense.

**Comment by shyamnamboodiripad** [comment]
:+1:

**Pull request merged**